lib/as_ocamlfind.ml

```diff
@@ -72,16 +74,20 @@ let parse_lines (byte, native, pp) =
     native_incs; native_objs; native_link;
     pp_incs; pp_objs; }
 
-let pkg_lookups ocamlfind name =
+let pkg_lookups ocamlfind native name =
```

Comment:
Basically this [line](https://github.com/samoht/assemblage/blob/master/lib/as_part_unit.ml#L128-L129) should tuple the compiler and the corresponding `` `Target``. The latter must be added to the context [here](https://github.com/samoht/assemblage/pull/163#discussion-diff-24478042R70) and there is nothing to touch in `ocamlfind.ml`.